README.md

```diff
@@ -14,575 +14,6 @@
-[2019-09-04 13:44:23] INFO: Purging backup 2019082604...
-[2019-09-04 13:44:23] INFO: Purging backup 2019082605...
-[2019-09-04 13:44:23] INFO: Purging backup 2019082606...
-[2019-09-04 13:44:24] INFO: Purging backup 2019082607...
-[2019-09-04 13:44:24] INFO: Purging backup 2019082608...
-[2019-09-04 13:44:24] INFO: Purging backup 2019082609...
-[2019-09-04 13:44:25] INFO: Purging backup 2019082610...
-[2019-09-04 13:44:25] INFO: Purging backup 2019082611...
-[2019-09-04 13:44:25] INFO: Purging backup 2019082612...
-[2019-09-04 13:44:26] INFO: Purging backup 2019082613...
-[2019-09-04 13:44:26] INFO: Cleaning up orphaned files...
-[2019-09-04 13:45:59] INFO: Purged 652 objects with a total size of 3.74 MB
-
-```
-
-Since SSTables and meta files are stored in different places for differential backups, the purge is a two step process:
-
-* Delete all backup directories
-* Scan active backup files from manifests and compare with the list of SSTables in the `data` directory. All SSTables present in the `data` directory but absent from all manifests will get deleted in that step.
-
-
-Check the status of a backup
-----------------------------
-
-```
-$ medusa status --help
-Usage: medusa status [OPTIONS]
-
-  Show status of backups
-
-Options:
-  --backup-name TEXT  Backup name  [required]
-  --help              Show this message and exit.
-```
-
-Outputs a summary of a specific backup status:
-
-```
-$ medusa status --backup-name=2019090503
-2019090503
-- Started: 2019-09-05 03:53:04, Finished: 2019-09-05 04:49:52
-- 32 nodes completed, 0 nodes incomplete, 0 nodes missing
-- 163256 files, 12.20 TB
-```
-
-
-Display information on the latest backup
------------------------------------------
-```
-$ medusa report-last-backup --help
-Usage: medusa report-last-backup [OPTIONS]
-
-  Find time since last backup and print it to stdout :return:
-
-Options:
-  --push-metrics  Also push the information via metrics
-  --help          Show this message and exit.
-
-```
-
-This command will display several informations on the latest backup:
-
-```
-$ medusa report-last-backup
-[2019-09-04 12:56:15] INFO: Latest node backup finished 18746 seconds ago
-[2019-09-04 12:56:18] INFO: Latest complete backup:
-[2019-09-04 12:56:18] INFO: - Name: 2019090407
-[2019-09-04 12:56:18] INFO: - Finished: 18173 seconds ago
-[2019-09-04 12:56:19] INFO: Latest backup:
-[2019-09-04 12:56:19] INFO: - Name: 2019090407
-[2019-09-04 12:56:19] INFO: - Finished: True
-[2019-09-04 12:56:19] INFO: - Details - Node counts
-[2019-09-04 12:56:19] INFO: - Complete backup: 180 nodes have completed the backup
-[2019-09-04 12:58:47] INFO: - Total size: 94.69 TiB
-[2019-09-04 12:58:55] INFO: - Total files: 5168096
-```
-
-When used with `--push-metrics`, Medusa will push completion metrics to the configured monitoring system.
+To get started with Medusa, backups and Restore for Cassandra, use the documentation in the [wiki of the project](../../wiki)
```

Comment:
> with Medusa, backups and Restore for Cassandra,

"backups" should have an uppercase B to be consistent with the rest of the text.

Reply:
I'd also trade `use` with `check`